### ISSUE TITLE
Bug Report: Endless Cookie Rendering Loop

### ISSUE THREAD

**Issue by Adebesin-Cell** [magiclabs/magic-js#733]
### ✅ Prerequisites

- [ ] Did you perform a cursory search of open issues? Is this bug already reported elsewhere?
- [ ] Are you running the latest SDK version?
- [ ] Are you reporting to the correct repository (`magic-sdk`)?

### 🐛 Description

Upon initializing the Wagmi connector into my project, I noticed a peculiar behavior related to cookie rendering.

### 🧩 Steps to Reproduce

1. Integrate the Wagmi connector into the project.
2. Initialize the Wagmi connector within the codebase.
3. Observe the rendering behavior of cookies.
4. Notice the continuous rendering of cookies without halt.

### 🤔 Expected behavior

I expected the cookies to render appropriately upon initialization and to cease rendering once initialized. There should be no continuous loop of cookie rendering.

### 😮 Actual behavior

The cookies enter an infinite rendering loop, continually rendering without cessation. This behavior triggers an error marked with number of times it has rendered, indicating that the cookies have been rendered an excessive number of times. Consequently, this error warns that third-party cookie functionality may be blocked.

### 💻 Code Sample

https://stackblitz.com/edit/stackblitz-starters-bbb65e?file=app%2Fpage.tsx,app%2Fconfig.ts,package.json

## Screenshots

![Screenshot 2024-03-25 at 5 38 25 PM](https://github.com/magiclabs/magic-js/assets/66301634/60e52ebc-24b9-4396-a64f-f69d02a21eb8)

### 🌎 Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| `magic-sdk`      | 18.3.0
| Browser          | Vivaldi
| `yarn`           | >=1.22.0
| Operating System | Mac OS Ventura 13.1 (22C65)

**Comment by Royal-lobster** [comment]
This seems to be a issue on almost all sites that has magic login integrations

![image](https://github.com/magiclabs/magic-js/assets/52039218/d503c395-7e64-4e0e-93c6-30c6be0aeff0)


<img width="1493" alt="image" src="https://github.com/magiclabs/magic-js/assets/52039218/96dae0fd-7de0-43bc-b30a-00de72535850">

<img width="1488" alt="image" src="https://github.com/magiclabs/magic-js/assets/52039218/d9253606-270b-47fb-af34-83b84a7cc123">

<img width="1743" alt="image" src="https://github.com/magiclabs/magic-js/assets/52039218/1b39fa35-eff6-4667-84e1-387a03a325e1">

**Comment by hubagaspar91** [comment]
Yep, we're experiencing this issue as well, we recently upgraded https://github.com/magiclabs/wagmi-magic-connector in our project from 1.1.5 to 2.0.0, which it seems also comes with a magic SDK version upgrade.

**Comment by TheGreatAlgo** [comment]
Having same issue too. we were on 8.1.0 (time for upgrade lol) and having the issue

**Comment by mjafri118** [comment]
is there a fix to this? upgrading does not seem to help.

**Comment by Wyxuch** [comment]
Hi there! 
this bug still occuring

**Comment by Bowis** [comment]
Same issue here still
